OpenDreamShared/Compiler/DM/DMParser.cs

```diff
@@ -1633,8 +1633,12 @@ class DMParser : Parser<Token> {
             DMValueType type = DMValueType.Anything;
 
             if (Check(TokenType.DM_As)) {
+                if (Check(TokenType.DM_LeftParenthesis) && Check(TokenType.DM_RightParenthesis))
```

Comment:
Whitespace needs to be checked for before the left parenthesis, and `Consume` should be used for the right parenthesis